README.md

```diff
@@ -54,8 +54,9 @@ module.exports = function(eleventyConfig) {
       clearScreen: false,
       server: {
         mode: "development",
-        middlewareMode: "ssr",
+        middlewareMode: "true",
```

Comment:
I think this should be a boooooolean and not a string right?

Reply:
Ooooops 😇 Yes, of course!

Reply:
This PR is a duplicate anyway: https://github.com/11ty/eleventy-plugin-vite/pull/12/commits/084acb360f095275e29e34cd7b922d8e9d21bc93 And I missed to adjust the default options in EleventyVite.js as well. You could also merge their PR and add the `appType: "custom",` line to README.md.

Reply:
Merged #12! Thank you!